mutiny-core/src/nostr/mod.rs

```diff
@@ -60,6 +78,8 @@ pub struct NostrManager<S: MutinyStorage> {
     pub storage: S,
     /// Lock for pending nwc invoices
     pending_nwc_lock: Arc<Mutex<()>>,
+    // Versioned nwc profiles for VSS. Option for backwards compatability.
+    nwc_profile_version: Option<NwcProfileVersion>,
```

Comment:
When this is `None` it is never set to be `Some` after it is updated. Probably what would be best is to not have this be optional and on init, if we don't have it, create a new one with version=0